admin/includes/extra_datafiles/dist-site_specific_admin_overrides.php

```diff
@@ -17,9 +17,13 @@
 // admin/invoice.php and admin/packingslip.php 
 // to determine whether attribute images should be displayed. 
 //
-// true ...... attribute images are shown on invoice and packingslip (the default) 
-// false ..... attribute images are not shown on invoice and packingslip 
+// $show_attrib_images   $show_attrib_images_pack
+// true or unset         true or unset            ...... attribute images are shown on invoice and packingslip (the default) 
+// false                 false of unset           ...... attribute images are NOT shown on invoice or packingslip 
```

Comment:
Change "false _of_ unset" to "false **or** unset".

Reply:
Good spot not picky!